.github/workflows/release.yml

```diff
@@ -52,26 +52,19 @@ jobs:
         name: Install signore
         uses: hashicorp/setup-signore-package@v1
       -
-        uses: hashicorp/setup-hc-releases@v1
+        name: Setup hc-releases
+        uses: hashicorp/actions-setup-hc-releases@v1
```

Comment:
PR is looking great!! Awesome job. Just one note- This will need to point to the `v2` tag, which will grab our launch version (0.1.0) of the hc-releases CLI -- being published tmr!